Soundify/Soundify/Scenes/Home/HomeViewController.swift

```diff
@@ -9,4 +9,59 @@
 import UIKit
 
 final class HomeViewController: UIViewController {
+    
+    private var albums: [Album]?
+    
+    @IBOutlet private weak var tableView: UITableView!
+    
+    override func viewDidLoad() {
+        fetchListOfNewReleases()
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView?.register(UINib(nibName: Cell.homeTableView, bundle: nil),
+                            forCellReuseIdentifier: Cell.homeTableView)
+    }
+    
+    private func fetchListOfNewReleases() {
+        UserRepository.shared.getListOfNewReleases(limit: nil, offset: nil) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case .success(let result):
+                if let albums = result?.albums?.items {
+                    self.albums = albums
+                    DispatchQueue.main.async {
+                        self.tableView.reloadData()
+                    }
+                }
+            case .failure(let error):
+                self.showErrorAlert(message: error.debugDescription)
+            default:
+                break
+            }
+        }
+    }
+}
+
+extension HomeViewController: UITableViewDataSource, UITableViewDelegate  {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if let size = albums?.count {
+            return size
+        }
+        return 0
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: Cell.homeTableView)
```

Comment:
```suggestion
       guard let cell = tableView.dequeueReusableCell(withIdentifier: Cell.homeTableView) as? HomeTableViewCell else { return UITableViewCell() }
```
sửa thêm ở dưới theo guard nhé